### ISSUE TITLE
Contrast ratio is NaN:1

### ISSUE THREAD

**Issue by AntonOellerer** [pa11y/pa11y#544]
Hey,
First of all, thank you for this great library!
I am currently trying to adapt our project to the WCAG2AAA standard, and discovered the following error message:
```
 • This element has insufficient contrast at this conformance level. Expected a
   contrast ratio of at least 4.5:1, but text in this element has a contrast
   ratio of NaN:1.
   (#main > section:nth-child(2) > div > div > h1)
   <h1 class="title">Alpha version</h1>
```
What confuses me about this is the NaN:1 ratio.
Do you know what the reason for this could be?

Environment: 
```
Pa11y:      5.3.0
Node.js:    14.15.0
npm:        6.14.8
OS:         5.9.11-200.fc33.x86_64 (linux)
```
Best regards
Anton Oellerer

**Comment by AntonOellerer** [comment]
I fixed this by setting the color values of the title explicitely, we are using react with bulma as css framework.

**Comment by danyalaytekin** [comment]
A bit late, but thanks @AntonOellerer and I'm glad you got it working. I'm not sure whether this requires anything to change in Pa11y so I'll close this for now.
